kingpin/actors/aws/cloudformation.py

```diff
@@ -804,8 +838,10 @@ class Stack(CloudFormationBaseActor):
         'role_arn': (str, None,
                      'The Amazon IAM Role to use when executing the stack'),
         'template': (str, REQUIRED,
-                     'Path to the AWS CloudFormation File. http(s)://, '
+                     'Path to the AWS CloudFormation File. s3://, '
```

Comment:
Lets raise an exception if both `template` and `template_url` are supplied - we don't want ambiguous behavior here..